jenkins/Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Windows Agent IT pipeline, run manually.
```

Comment:
Having two files named "Jenkinsfile" could be confusing. Perhaps we can have Jenkinsfile.windows and Jenkinsfile.linux, both in the top level directory. The CI configuration allows you to specify the filename, so we're not limited to the "Jenkinsfile" naming convention.

Reply:
Done